src/Compilers/CSharp/Test/Symbol/Symbols/Metadata/PE/LoadingAttributes.cs

```diff
@@ -1316,7 +1316,8 @@ public void M1(decimal d1 = -7)
             });
         }
 
-        [Fact]
+        // Bug for getting this running on Mono: https://github.com/dotnet/roslyn/issues/23760
```

Comment:
Rather than a comment use the `[WorkItem]` attribute. This makes it easier for us to use tooling to look at issues related to our unit tests.